tests/test_plot_metrics.py

```diff
@@ -0,0 +1,39 @@
+import re
+from unittest.mock import MagicMock, Mock, patch
+
+import numpy as np
+import pytest
+
+from astrorapid.plot_metrics import plot_metrics
+
+def test_plot_metrics(monkeypatch):
+    """Test the basic plotting functionality"""
+    class_names = ('A', 'B')
+    X_test = np.zeros((500, 50, 4))
+    y_test = np.zeros((500, 50, len(class_names) + 1))
```

Comment:
@daniel-muthukrishna I modified the unittest to match the expected array dimensions that we were talking about over teams. The IndexError I was talking about no longer appears now.